amundsenatlastypes/kickstart.py

```diff
@@ -1,13 +1,21 @@
 import re
 
 from atlasclient.utils import parse_table_qualified_name
+from requests import Timeout
 
 from amundsenatlastypes.client import driver
 
 
 # noinspection SpellCheckingInspection
 class KickstartExistingData:
 
+    def create_entities(self, entities_to_create):
+        try:
+            driver.entity_bulk.create(data={"entities": entities_to_create})
+        except Timeout as ex:
+            print(f'ReadTimeout : {ex}')
+            self.create_entities(entities_to_create)
```

Comment:
what is the default timeout ? That can overflow the Atlas server.

Reply:
It's 10 seconds, but sometimes on the local instance of Atlas, it throws readout timeout on local machine most of the times. To avoid that, adding this try/except blocks.

Reply:
But probably should just fail after several retries. What do you think ?

Reply:
@dwarszawski makes sense. made it to only try one more time.